firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -388,9 +392,37 @@ uint8_t TCMDEXEC_heartbeat_on(const char *args_str, TCMD_TelecommandChannel_enum
 
 uint8_t TCMDEXEC_core_system_stats(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
                         char *response_output_buf, uint16_t response_output_buf_len) {
-    // TODO: implement this (Issue #103)
-    // Use `TCMD_get_agenda_used_slots_count`
-    snprintf(response_output_buf, response_output_buf_len, "System stats: TODO\n");
+    
+    uint32_t epoch = TIM_get_current_system_uptime_ms(); 
```

Comment:
Setting to variables is largely unnecessary. Just call the functions within sprintf to avoid having to match up another intermediate name.

If you do use variables like this, please set any variables which do not change (within the scope) to `const`.